docs/virtual_machines/istio_service_mesh.md

```diff
@@ -0,0 +1,92 @@
+# Istio service mesh
+
+Service mesh allows to monitor, visualize and control traffic between microservices.
+Kubevirt supports running VMs as a part of Istio service mesh.
+
+## Limitations
+
+- Istio service mesh is only supported with a pod network masquerade binding.
+
+- Istio uses a [list of ports](https://istio.io/latest/docs/ops/deployment/requirements/#ports-used-by-istio) for it's own purposes, these ports must not
+  be explicitly specified in a VMI interface.
+
+## Prerequisites
+
+This guide assumes that Istio is already deployed and uses Istio CNI Plugin. See [Istio documentation](https://istio.io/latest/docs/) for more information.
+
+## Create a VirtualMachineInstance with enabled Istio proxy injecton
+
+If your cluster uses Multus to manage CNI, the following NetworkAttachmentDefinition is required in the application namespace:
+
+```yaml
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: istio-cni
+```
+
+The example below specifies a VMI with masquerade network interface and annotation instructing
```

Comment:
Instead of "webhook to injext the proxy container" we could say "to register the VM into the service mesh" or something similar? I'm thinking how to make this closer to an end user who does not care about proxy containers being involved

Reply:
Updated accordingly to your suggestion